pkg/model/spotinstmodel/instance_group.go

```diff
@@ -574,7 +586,7 @@ func (b *InstanceGroupModelBuilder) buildRootVolumeOpts(ig *kops.InstanceGroup)
 	{
 		typ := fi.StringValue(ig.Spec.RootVolumeType)
 		if typ == "" {
-			typ = awsmodel.DefaultVolumeType
+			typ = "gp2"
```

Comment:
Is there no other compatibility within spotinst? Hardcoding defaults looks weird to me - why is this an option to set then?

Reply:
We just follow the [AWS implementation](https://github.com/kubernetes/kops/blob/d1d41520a519717fcd3575f2e313efad889995e9/pkg/model/awsmodel/autoscalinggroup.go#L150-L153), but have to replace the use of the const `awsmodel.DefaultVolumeType` with its value to avoid cyclic imports (since the AWS model imports the Spotinst one, see: [autoscalinggroup.go#L28](https://github.com/kubernetes/kops/blob/d1d41520a519717fcd3575f2e313efad889995e9/pkg/model/awsmodel/autoscalinggroup.go#L28) and [autoscalinggroup.go#L58-L63](https://github.com/kubernetes/kops/blob/d1d41520a519717fcd3575f2e313efad889995e9/pkg/model/awsmodel/autoscalinggroup.go#L58-L63)). 

Do you think we should implement it differently? If so, I'd change it.

Reply:
Oh this is just about cyclic imports, I see. Thanks!